File: app/src/main/java/com/vn/ezlearn/models/Question.java
package com.vn.ezlearn.models;

import java.util.List;

/**
 * Created by FRAMGIA\nguyen.duc.manh on 10/10/2017.
 */

public class Question {
    public static final int TYPE_QUESTION = 1;
    public static final int TYPE_READING = 2;
    public Region region;
    public Integer type;
    public List<Content> question;
    public List<Reading> reading;
}

File: app/src/main/java/com/vn/ezlearn/models/Question.kt
package com.vn.ezlearn.models

/**
 * Created by FRAMGIA\nguyen.duc.manh on 10/10/2017.
 */

class Question {
    var region: Region? = null
    var type: Int? = null
    var question: List<Content>? = null
    var reading: List<Reading>? = null

    companion object {
        val TYPE_QUESTION = 1
        val TYPE_READING = 2
    }
}
